.github/tpl/build.tpl.yml

```diff
@@ -32,6 +32,7 @@ concurrency:
 jobs:
   build:
     strategy:
+      max-parallel: 2
```

Comment:
why?

Reply:
This one is a weird one that I can't fully explain.  So whenever 2 are run in parallel, networking seems to always be fine.  But when 4 are run in parallel, you'll get one node that for whatever reason can't perform name resolutions at all.  I don't know if somehow there is some weird ip address conflict or something, I wouldn't think that would be possible though as I thought the default qemu network used NAT.  I did add in some debug to show the network config in postBuild.sh in case it happens with 2 simultaneous builds so we can see if maybe the /etc/resolv.conf doesn't get a nameserver populated at all.

Reply:
btw, i previously thought this was an issue with OpenCSW but from other commands run I could see it was the entire name resolution even to other places...

Reply:
then remove it.

Reply:
ok, i'll try removing it again, but pretty sure one will fail due to network issues